webapp-django/crashstats/crashstats/fixtures/products.json

```diff
@@ -0,0 +1,83 @@
+[
+  {
+    "model": "crashstats.product",
+    "pk": 1,
+    "fields": {
+      "product_name": "Firefox",
+      "sort": 1,
+      "is_active": true
+    }
+  },
+  {
+    "model": "crashstats.product",
+    "pk": 2,
+    "fields": {
+      "product_name": "FennecAndroid",
+      "sort": 2,
+      "is_active": true
+    }
+  },
+  {
+    "model": "crashstats.product",
+    "pk": 3,
+    "fields": {
+      "product_name": "Focus",
+      "sort": 3,
+      "is_active": true
+    }
+  },
+  {
+    "model": "crashstats.product",
+    "pk": 4,
+    "fields": {
+      "product_name": "FirefoxReality",
+      "sort": 4,
+      "is_active": true
+    }
+  },
+  {
+    "model": "crashstats.product",
+    "pk": 5,
+    "fields": {
+      "product_name": "GeckoViewExample",
+      "sort": 11,
+      "is_active": true
+    }
+  },
+  {
+    "model": "crashstats.product",
+    "pk": 6,
+    "fields": {
+      "product_name": "ReferenceBrowser",
+      "sort": 12,
+      "is_active": true
+    }
+  },
+  {
+    "model": "crashstats.product",
+    "pk": 7,
+    "fields": {
+      "product_name": "Thunderbird",
+      "sort": 80,
+      "is_active": true
+    }
+  },
+  {
+    "model": "crashstats.product",
+    "pk": 8,
+    "fields": {
+      "product_name": "SeaMonkey",
+      "sort": 90,
+      "is_active": true
+    }
+  },
+  {
+    "model": "crashstats.product",
+    "pk": 9,
+    "fields": {
+      "product_name": "Fennec",
+      "sort": 100,
+      "is_active": true
+    }
+  }
+]
```

Comment:
I don't know if it makes sense to have the fixture have all the current products, but I didn't want to think about it much so I just did it this way.

This only affects the local development environment.